pandas/core/frame.py

```diff
@@ -2412,11 +2419,43 @@ def eval(self, expr, inplace=False, **kwargs):
 
         Examples
         --------
-        >>> from numpy.random import randn
-        >>> from pandas import DataFrame
-        >>> df = pd.DataFrame(randn(10, 2), columns=list('ab'))
-        >>> df.eval('a + b')
-        >>> df.eval('c = a + b')
+        >>> df = pd.DataFrame({'A': range(1, 6), 'B': range(10, 0, -2)})
+        >>> df
+           A   B
+        0  1  10
+        1  2   8
+        2  3   6
+        3  4   4
+        4  5   2
+        >>> df.eval('A + B')
+        0    11
+        1    10
+        2     9
+        3     8
+        4     7
+        dtype: int64
+        >>> df.eval('C = A + B')
```

Comment:
Could you add a line before this saying "Assignment is allowed and by default the original DataFrame is not modiefied."